source/_docs/blueprint/selectors.markdown

```diff
@@ -18,24 +18,29 @@ The following selectors are currently available:
 - [Action selector](#action-selector)
 - [Add-on selector](#add-on-selector)
 - [Area selector](#area-selector)
+  - [Example area selectors](#example-area-selectors)
```

Comment:
This is not a TOC. Please revert these changes.

Reply:
Done.  I didn't even realize it did that.  The `markdown.extension.toc.updateOnSave` setting forced it.

Reply:
Yeah, I figured, that setting can be annoying at times